crates/libwebrtc-sys/build.rs

```diff
@@ -70,13 +79,31 @@ fn main() -> anyhow::Result<()> {
 /// Downloads and unpacks compiled `libwebrtc` library.
 fn download_libwebrtc() -> anyhow::Result<()> {
     let mut libwebrtc_url = env::var("LIBWEBRTC_URL")?;
-    libwebrtc_url.push_str("/libwebrtc-win-x64.tar.gz");
-
+    libwebrtc_url.push('/');
     let manifest_path = PathBuf::from(env::var("CARGO_MANIFEST_DIR")?);
     let temp_dir = manifest_path.join("temp");
-    let archive = temp_dir.join("libwebrtc-win-x64.tar.gz");
     let lib_dir = manifest_path.join("lib");
 
+    let archive_name = {
+        let mut archive_name = String::from("libwebrtc-");
+        #[cfg(target_os = "windows")]
+        archive_name.push_str("windows-");
+        #[cfg(target_os = "linux")]
+        archive_name.push_str("linux-");
+        #[cfg(target_os = "macos")]
+        archive_name.push_str("macos-");
+
+        #[cfg(target_arch = "aarch64")]
+        archive_name.push_str("arm64.tar.gz");
+        #[cfg(target_arch = "x86_64")]
+        archive_name.push_str("x64.tar.gz");
+
+        archive_name
+    };
+
+    libwebrtc_url.push_str(archive_name.as_str());
+    let archive = temp_dir.join(archive_name);
```

Comment:
We use `process::Command::new("tar")` below. Some distributions like Windows may miss this command in some setups. It would be nice to use some crate for unpacking `.tar.gz` as `[build-dependencies]`.